ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op_sharded_program_factory.cpp

```diff
@@ -1027,6 +1047,7 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_sharded_v2_impl(
         // TODO: Generalize this to not make this assumption
         read_window_in_inner_loop = true;
         if (!block_sharded) {
+            TT_THROW("Weight width sliced only supported for block sharded conv");
```

Comment:
this doesn't make sense, if this code is illegal, than code in this branch below the throw should be deleted.
Seems like you want TT_FATAL(block_sharded, "... ")

Reply:
I added that code to ensure that this codepath is actually never used. It can be removed then.

Reply:
Removed.